lua/null-ls/builtins/formatting/npm_groovy_lint.lua

```diff
@@ -13,7 +13,8 @@ return h.make_builtin({
     filetypes = { "groovy", "java", "Jenkinsfile" },
     generator_opts = {
         command = "npm-groovy-lint",
-        args = { "--format", "-" },
+        args = { "--failon", "none", "--format", "-" },
```

Comment:
This is fine but please kept PR changes atomic if you can @mosheavni

Reply:
huh? it's not on this PR